### PR TITLE
Stop fuse web server

### PR DESCRIPTION
**Pull request by huanghua78**
And refactor fuse start code into startCommon() and stopCommon()

### What changes are proposed in this pull request?

Stop Fuse web server when exit.

### Why are the changes needed?

Fuse web server must be stopped, otherwise the main thread is waiting for it forever.

### Does this PR introduce any user facing changes?

N/A

### CODE REVIEW

File: dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java [hunk 1]
```diff
@@ -64,36 +64,31 @@
 @ThreadSafe
 public class AlluxioFuse {
   private static final Logger LOG = LoggerFactory.getLogger(AlluxioFuse.class);
+  private FuseWebServer mWebServer = null;
```
Comment (dbw9580) [inline dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java:67]:
```suggestion
  @Nullable
  private FuseWebServer mWebServer = null;
```

File: dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java [hunk 2]
```diff
@@ -64,36 +64,31 @@
...
-    FuseOptions fuseOptions = FuseOptions.Builder.fromConfig(conf).build();
-
-    FileSystemContext fsContext = FileSystemContext.create(conf);
-    if (!fuseOptions.getFileSystemOptions().getUfsFileSystemOptions().isPresent()
-        && !fuseOptions.getFileSystemOptions().isDoraCacheEnabled()) {
-      // cases other than standalone fuse sdk
-      conf = AlluxioFuseUtils.tryLoadingConfigFromMaster(fsContext);
-    }
-
+  protected void start_common(AlluxioConfiguration conf,
```
Comment (dbw9580) [inline dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java:80]:
```suggestion
  protected void startCommon(AlluxioConfiguration conf,
```

File: dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java [hunk 3]
```diff
@@ -117,6 +112,38 @@ public void start(AlluxioConfiguration conf) throws ParseException {
     }
   }
 
+  protected void stop_common() {
```
Comment (dbw9580) [inline dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java:115]:
```suggestion
  protected void stopCommon() {
```

**Review by dbw9580 — APPROVED** [review]
LGTM with a few minor comments

File: dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java (hunk 1 above)
Comment (huanghua78) [reply dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java:70]: Done.

File: dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java (hunk 2 above)
Comment (huanghua78) [reply dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java]: Done

File: dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java (hunk 3 above)
Comment (huanghua78) [reply dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java]: Done

**Comment by huanghua78** [comment]
alluxio-bot, merge this please

**Pull request merged**